### ISSUE TITLE
Top nav bar changes color on scroll

### ISSUE THREAD

**Issue by machinaeZER0** [hjiangsu/thunder#196]
I just noticed that in (I believe) all themes, the top nav bar (with the hamburger menu and sorting options) is a certain color at the top of a feed/page, which immediately changes to a different color as you scroll any distance down a feed/page. Scrolling back up to the top of any page sets the bar back to its original color

This is on Android - not sure at this point if it happens on iOS or not. Figure there's a nonzero chance this is intentional, but figured I'd log it just to be sure!

**Comment by hjiangsu** [comment]
This is due to material 3 design, which changes the colour of the top nav bar when scrolling occurs! This gives you a visual cue that you are not at the top of the page

**Comment by machinaeZER0** [comment]
Oh. Huh! Interesting :) Nevermind then!

**Comment by hjiangsu** [comment]
Yep! You can also see this in this demo: https://flutter.github.io/samples/web/material_3_demo/#/

Notice how the top bar changes colour when you scroll

**Comment by machinaeZER0** [comment]
It's funny to think about all the little things that go into a design language that people may not even notice :D That's neat!
